.github/workflows/ci.yml

```
@@ -106,6 +106,11 @@ jobs:
         with:
           node-version: "20"
 
+      - name: Setup Python 3.10
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)